### ISSUE TITLE
Windows system requirements need to mention Intel

### ISSUE THREAD

**Issue by stuartmorgan** [flutter/flutter#53306]
Per #53120, Flutter currently doesn't work on ARM-based Windows 10 machines (e.g., the Yoga C630). Currently our system requirements for Windows just say "Windows 7 SP1 or later (64-bit)"; these machine meet those requirements, but won't actually allow Flutter development.

**Comment by therealansh** [comment]
I'll take up this issue and add the system requirements.

**Comment by iapicca** [comment]
I'd say X86-64 rather then Intel,
AMD is quite ubiquitous right now

**Comment by sfshaza2** [comment]
If this is exclusively a flutter.dev docs issue (and it looks like it might be), let's move this to the website repo.

**Comment by stuartmorgan** [comment]
@sfshaza2 Yes, I was thinking specifically of https://flutter.dev/docs/get-started/install/windows, so feel free to move it.
